scanner/enricher/csaf/csaf.go

```diff
@@ -0,0 +1,203 @@
+// Package csaf provides a Red Hat CSAF enricher.
+// The contents are strongly based on https://github.com/quay/claircore/tree/v1.5.34/rhel/vex.
+//
+// This exists as a temporary solution at this point, but there is potential for repurposing.
+// TODO(ROX-26672): This enricher may no longer be needed once this is done.
+package csaf
```

Comment:
nit / not for this PR: Love the package level comment, during review I looked at this file last :( , perhaps adopting the `doc.go` convention would be beneficial throughout the codebase. /randomthoughts

Reply:
Found some random examples:

https://github.com/golang/go/blob/master/src/fmt/doc.go

https://github.com/golang/go/blob/master/src/bufio/bufio.go